framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -305,6 +306,8 @@ def index(self):
                                                           last_request_type=cherrypy.request.method,
                                                           is_first_fragment=True)
 
+        common.add2session(KEY_ADAPTER, '13')
```

Comment:
This should not be ...

Reply:
Yeah, I intented to eliminate that, thanks for the reminder.

Reply:
You resolved this, but the undesired change seem to be still here...